cime_config/testlist_allactive.xml

```diff
@@ -275,14 +266,13 @@
       <option name="wallclock"> 00:30 </option>
     </options>
   </test>
-  <test name="NCK_Ld5" grid="f19_g17" compset="B1850GWs" testmods="allactive/cism/test_coupling">
```

Comment:
For this and some other NCK and _N3 tests: see the comment in the xml file a few lines down: I think we need to keep at least one stub wav compset for testing unless ww3 now supports multi-instance.

Reply:
From the initial description of this PR: WW3 supports multi-instance now

Reply:
Oops, sorry, I guess I should read more!